source/opt/def_use_manager.h

```diff
@@ -121,7 +97,14 @@ class DefUseManager {
   // Erases the records that a given instruction uses its operand ids.
   void EraseUseRecordsOfOperandIds(const ir::Instruction* inst);
 
-  const MessageConsumer& consumer_;  // Message consumer.
+ private:
+  using InstToUsedIdsMap =
+      std::unordered_map<const ir::Instruction*, std::vector<uint32_t>>;
+
+  // Analyzes the defs and uses in the given |module| and populates data
+  // structures in this class. Does nothing if |module| is nullptr.
+  void AnalyzeDefUse(ir::Module* module);
+
   IdToDefMap id_to_def_;             // Mapping from ids to their definitions
   IdToUsesMap id_to_uses_;           // Mapping from ids to their uses
   // Mapping from instructions to the ids used in the instructions generating
```

Comment:
Not part of this CL, but I'm confused by this comment.

Reply:
Fixed up.  From other comments I think it was originally a map from and id to the ids used in the instruction that defines the id.